site/reference/pkg/cat/README.md

```diff
@@ -10,77 +10,52 @@ description: >
     Print the resources in a file/directory
 -->
 
-Prints the resources in a file/directory as yaml to stdout.
-
-`cat` is useful for printing the resources in a file/directory which might
-contain other non-resource files.
+`cat` prints the resources in a file or directory as yaml to stdout.
```

Comment:
@phanimarupaka where did we land on with `tree` vs ` cat` vs `view`? Also, more generally with all commands being verbs.

Reply:
I feel we should retain `tree` name and deprecate `cat` for now. We din't see strong use-cases for `cat`. We can always add it later if we want to. WDYT?

Reply:
`status` as in live status ? I feel we should retain that too as it is highly intuitive and common word.